src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -209,8 +209,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       Overwrite="true" />
 
     <!-- Requires Windows 10 version 1803 or newer -->
-    <Message Importance="High" Text="Executing: $(_TarCommand) -czf $(_TarArchiveOutputPath) ." />
-    <Exec Command="$(_TarCommand) -czf $(_TarArchiveOutputPath) ."
+    <Message Importance="High" Text="Executing: $(_TarCommand) -czf &quot;$(_TarArchiveOutputPath)&quot; ." />
+    <Exec Command="$(_TarCommand) -czf &quot;$(_TarArchiveOutputPath)&quot; ."
```

Comment:
Does the command need quotes?

Reply:
Slight corner case but, nonetheless, I'm double-checking this one as well as a few more of our eng/scripts scripts&hellip;